### ISSUE TITLE
Crash if a Pokemob breaks out of a Pokeball

### ISSUE THREAD

**Issue by WenXin20** [Pokecube-Development/Pokecube-Issues-and-Wiki#117]
#### Issue Description:
In my small custom modpack, if a pokemob breaks out of the pokeball while trying to catch it, I get a crash. This happened using the regular red and white pokeball.
[Crash Report](https://gist.github.com/WenXin20/3a37aab0cb536a546fdc4d8d55c86ef4)

#### What happens:
If a pokemob breaks out of the pokeball while trying to catch it, I get a crash with the following error at the top of the report: `java.lang.IllegalStateException: Entity is already tracked!`

#### What you expected to happen:
The pokemob should break out without crashing the game

#### Steps to reproduce:

1. Find a pokemob.
2. Catch it.
3. Crash if it breaks out successfully.
...

____
#### Affected Versions:

- Pokecube AIO: v2.0.a
- Minecraft: v1.14.4
- Forge: v28.1.87

**Comment by Audalics** [comment]
Still happening
```
[22Dec2019 12:55:32.820] [Client thread/INFO] [net.minecraft.client.gui.NewChatGui/]: [CHAT] Pikachu broke out!
[22Dec2019 12:55:32.822] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.822] [Server thread/ERROR] [net.minecraft.server.MinecraftServer/]: Encountered an unexpected exception
net.minecraft.crash.ReportedException: Exception ticking world
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:833) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:764) ~[?:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:112) ~[?:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:622) [?:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_232]
Caused by: java.lang.IllegalStateException: Entity is already tracked!
	at net.minecraft.world.server.ChunkManager.func_219210_a(ChunkManager.java:923) ~[?:?]
	at net.minecraft.world.server.ServerChunkProvider.func_217230_c(SourceFile:485) ~[?:?]
	at net.minecraft.world.server.ServerWorld.func_217465_m(ServerWorld.java:979) ~[?:?]
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:363) ~[?:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:829) ~[?:?]
	... 4 more
[22Dec2019 12:55:32.824] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.824] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.825] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.825] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.825] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.825] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.825] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.826] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.826] [Client thread/INFO] [STDOUT/]: [pokecube.core.interfaces.capabilities.impl.PokemobGenes:setSize:499]: EntityDimensions w=0.4, h=0.4, fixed=true 1.0 0.4 0.4 0.4
[22Dec2019 12:55:32.826] [Client thread/INFO] [STDOUT/]: [WolfShotz.Wyrmroost.event.EventHandler$Client:ridingPerspective:121]: Fired
[22Dec2019 12:55:32.829] [Server thread/ERROR] [net.minecraft.server.MinecraftServer/]: This crash report has been saved to: /home/alexander/.config/GDLauncher/GDLauncher/instances/packs/KISS/crash-reports/crash-2019-12-22_12.55.32-server.txt
[22Dec2019 12:55:32.831] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Stopping server
[22Dec2019 12:55:32.834] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving players
[22Dec2019 12:55:32.868] [Client thread/INFO] [STDOUT/]: [net.minecraft.util.registry.Bootstrap:func_179870_a:100]: ---- Minecraft Crash Report ----
// But it works on my machine.

Time: 12/22/19 12:55 PM
Description: Exception ticking world

java.lang.IllegalStateException: Entity is already tracked!
	at net.minecraft.world.server.ChunkManager.func_219210_a(ChunkManager.java:923) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.server.ServerChunkProvider.func_217230_c(SourceFile:485) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.server.ServerWorld.func_217465_m(ServerWorld.java:979) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:fml:quark:change-sleeping-player-count,xf:fml:quark:add-rave-hook}
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:363) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:fml:quark:change-sleeping-player-count,xf:fml:quark:add-rave-hook}
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:829) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:764) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:112) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:622) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_232] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at net.minecraft.world.server.ChunkManager.func_219210_a(ChunkManager.java:923)
	at net.minecraft.world.server.ServerChunkProvider.func_217230_c(SourceFile:485)
	at net.minecraft.world.server.ServerWorld.func_217465_m(ServerWorld.java:979)
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:363)

-- Affected level --
Details:
	All players: 1 total; [ServerPlayerEntity['Audalics'/198, l='New World', x=-154.59, y=29.00, z=100.91]]
	Chunk stats: ServerChunkCache: 2246
	Level dimension: DimensionType{minecraft:overworld}
	Level name: New World
	Level seed: -6950633211841124742
	Level generator: ID 07 - biomesoplenty, ver 0. Features enabled: true
	Level generator options: {}
	Level spawn location: World: (-144,63,43), Chunk: (at 0,3,11 in -9,2; contains blocks -144,0,32 to -129,255,47), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 91314 game time, 100713 day time
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 144787 (now: false), thunder time: 20124 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:829)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:764)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:112)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:622)
	at java.lang.Thread.run(Thread.java:748)

-- System Details --
Details:
	Minecraft Version: 1.14.4
	Minecraft Version ID: 1.14.4
	Operating System: Linux (amd64) version 4.15.0-72-generic
	Java Version: 1.8.0_232, AdoptOpenJDK
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), AdoptOpenJDK
	Memory: 5202733704 bytes (4961 MB) / 11252793344 bytes (10731 MB) up to 22429564928 bytes (21390 MB)
	CPUs: 8
	JVM Flags: 2 total; -Xms256m -Xmx24064m
	ModLauncher: 4.1.0+62+5bfa59b
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/eventbus-1.0.0-service.jar eventbus PLUGINSERVICE 
		/forge-1.14.4-28.1.107.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.14.4-28.1.107.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-1.0.1-milestone.0.1+94458e7-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.14.4-28.1.107.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.14.4-28.1.107.jar runtimedistcleaner PLUGINSERVICE 
		/forge-1.14.4-28.1.107.jar fml TRANSFORMATIONSERVICE 
	FML: 28.1
	Forge: net.minecraftforge:28.1.107
	FML Language Providers: 
		javafml@28.1
		minecraft@1
		kotlinfml@1.4.0
	Mod List: 
		tetra-1.14.4-1.2.1.jar Tetra {tetra@1.14.4-1.2.1 DONE}
		CyclopsCore-1.14.4-1.6.1.jar Cyclops Core {cyclopscore@1.6.1 DONE}
		doubledoors_1.14-1.2.jar Double Doors {doubledoors@1.2 DONE}
		extendedfoods-1.14.4-0.0.2-beta.jar Extended Foods {extendedfoods@0.0.2-beta DONE}
		Cucumber-1.14.4-2.0.6.jar Cucumber Library {cucumber@2.0.6 DONE}
		Wyrmroost-1.14.4-1.0.2.jar Wyrmroost {wyrmroost@1.0.2 DONE}
		nethercraft-1.14.4-1.0.3.jar Nethercraft {nethercraft@1.0.3 DONE}
		bountifulbaubles-1.14.4-0.0.2.jar Bountiful Baubles {bountifulbaubles@NONE DONE}
		Mine and Slash-1.14.4-2.9.4.jar Mine and Slash {mmorpg@2.8.0 DONE}
		jei-1.14.4-6.0.0.26.jar Just Enough Items {jei@6.0.0.26 DONE}
		extendedslabs-1.14.4-0.1.0-beta.jar Extended Slabs {extendedslabs@0.1.0-beta DONE}
		Druidcraft-1.14-0.2.32.jar Druidcraft {druidcraft@0.2.32 DONE}
		Uppers-0.1.10.jar Uppers {uppers@0.1.10 DONE}
		extrabows-v1.14.4 b6.jar Extra Bows {extrabows@v1.14.4 b6 DONE}
		EditSign-1.14.4-1.0.1+4.jar Edit Sign {edit_sign@1.0.1+4 DONE}
		Upgrade-Aquatic-1.14.4-v1.4.2.jar Upgrade Aquatic {upgrade_aquatic@1.4.2 DONE}
		modnametooltip_1.14.3-1.12.1.jar Mod Name Tooltip {modnametooltip@1.12.1 DONE}
		Mine and Slash Auto Compatibility 1.14.4-1.4.7.jar M&S Auto-Compat 1.14.4 {azurecompat@1.4.7 DONE}
		BetterCaves-1.14.4-1.1.1.jar YUNG's Better Caves {bettercaves@1.14.4-1.1.1 DONE}
		yoyos_1.14.4-2.0.1.5.jar Yoyos {yoyos@2.0.1.5 DONE}
		wsw_1.14-1.2.jar Weaker Spiderwebs {wsw@1.2 DONE}
		glassential-forge-1.14.4-1.1.1.jar Glassential {glassential@1.1.1 DONE}
		ForgeEndertech-1.14.4-5.0.0.0-build.0040.jar Forge Endertech {forgeendertech@1.14.4-5.0.0.0 DONE}
		XaerosWorldMap_1.5.2_Forge_1.14.4.jar Xaero's World Map {xaeroworldmap@1.5.2 DONE}
		OresAboveDiamonds-1.14.4 v1.3Beta.jar Ores Above Diamonds {oresabovediamonds@1.3b DONE}
		mgui-1.14.4-1.0.1.jar mgui {mgui@1.0.1 DONE}
		CookingForBlockheads_1.14.4-7.4.1.jar Cooking for Blockheads {cookingforblockheads@7.4.1 DONE}
		Controlling-5.0.4.jar Controlling {controlling@5.0.4 DONE}
		Placebo-1.14.4-2.3.5.jar Placebo {placebo@2.3.5 DONE}
		citadel-1.0.0.jar Citadel {citadel@1.0.0 DONE}
		SereneSeasons-1.14.4-2.0.0.9-universal.jar Serene Seasons {sereneseasons@1.14.4-2.0.0.9 DONE}
		SkeletonLamps1.01-1.14.4.jar Skeleton Lamps (Forthteen Edition) {skltlampforthteen@1.01 (Minor Patches #1) DONE}
		VanillaTweaks-1.14.4.1.5.11.jar VanillaTweaks {vanillatweaks@1.14.4.1.5.11 DONE}
		betteranimalsplus-1.14.4-8.0.0.jar Better Animals Plus {betteranimalsplus@8.0.0 DONE}
		WitherSkeletonTweaks-1.14.4-3.1.0.jar Wither Skeleton Tweaks {wstweaks@3.1.0 DONE}
		PickUpNotifier-v1.1.1-1.14.4.jar Pick Up Notifier {pickupnotifier@1.1.1 DONE}
		mysticalworld-1.14.4-1.6.12.jar Mystical World {mysticalworld@1.14.4-1.6.12 DONE}
		GemsPlusPlus-Forge-1.14.4-1.4.0.jar GemsPlusPlus {gemsplusplus@V1.4.0 DONE}
		forge-1.14.4-28.1.107-universal.jar Forge {forge@28.1.107 DONE}
		mysticallib-1.14.4-1.10.0.jar Mystical Lib {mysticallib@1.14.4-1.10.0 DONE}
		PattysMoreStuff-Stable-1.14.4-1.2.2.jar pattysmorestuff {pattysmorestuff@1.2.2 DONE}
		moveplus-1.14.4-2.4.6.jar Move Plus {moveplus@2.4.6 DONE}
		Explorercraft-1.14.4-3.2.4.jar Explorercraft {explorercraft@3.2.4 DONE}
		Mine and Decoration-1.0.6.6.jar Mine & Decoration {mineanddecoration@1.0.6.6 DONE}
		spw_1.14-1.0.jar Spiders Produce Webs {spw@1.0 DONE}
		SimpleDivingGear-3.0.3 mc 1.14.4.jar Simple Diving Gear {simpledivegear@3.0.3 mc 1.14.4 DONE}
		CraftingTweaks_1.14.4-10.1.4.jar Crafting Tweaks {craftingtweaks@10.1.4 DONE}
		UselessSword-1.14.4-V1.0.1.jar Useless Sword {useless_sword@1.0.0 DONE}
		forge-1.14.4-28.1.107-client.jar Minecraft {minecraft@1.14.4 DONE}
		Craft_and_Hunt_V0.9.3_MC_1.14.4.jar Craft and Hunt {craftandhunt@0.4 DONE}
		mcw-bridges-1.0.2-mc1.14.4.jar Macaw's Bridges {mcwbridges@1.0.2 DONE}
		Flopper-1.14.4-1.0.5.jar Flopper {flopper@1.0.5 DONE}
		PattysMoreTools-2-1.14.4-2.4.jar PattysMoreTools-2 {pattysmoretools@2.4 DONE}
		theoneprobe-1.14-1.4.37.jar The One Probe {theoneprobe@NONE DONE}
		MouseTweaks-2.12-mc1.14.4.jar Mouse Tweaks {mousetweaks@2.12 DONE}
		EasySteel-1.14.4-1.0.51.jar Easy Steel & More {easy_steel@1.0.51 DONE}
		UndergroundBiomes-1.14.4-1.0-53.jar Underground Biomes {undergroundbiomes@1.0-53 DONE}
		BiomesOPlenty-1.14.4-9.0.0.261-universal.jar Biomes O' Plenty {biomesoplenty@1.14.4-9.0.0.259 DONE}
		behgameon-1.14.4-1.0.1.jar Behgameon {behgameon@1.0.1 DONE}
		bluepower-1.14.4-0.3.161-alpha-universal.jar Blue Power {bluepower@0.3.161-alpha DONE}
		C.O.M.B.A.T.-1.14.4-1.0.2.jar C.O.M.B.A.T. {combat@1.0.2 DONE}
		xercamod-1.14.4-1.3.jar Xerca Mod {xercamod@1.14.4-1.3 DONE}
		paintings-1.14.4-6.0.1.5.jar Paintings ++ {paintings@1.14.4-6.0.1.5 DONE}
		curios-FORGE-1.14.4-1.0.6.jar Curios API {curios@FORGE-1.14.4-1.0.6 DONE}
		SpartanShields-1.14.4-2.0.0.jar Spartan Shields {spartanshields@2.0.0 DONE}
		Xaeros_Minimap_1.19.2_Forge_1.14.4.jar Xaero's Minimap {xaerominimap@1.19.2 DONE}
		PeacefulSurface-1.14.4-v1.jar PeacefulSurface {peacefulsurface@1.14.4-v1 DONE}
		AutoRegLib-1.4-35.jar AutoRegLib {arl@1.4-35 DONE}
		Quark-r2.0-202.jar Quark {quark@r2.0-202 DONE}
		rediscovered-1.14.4-1.1.1.jar Rediscovered {rediscovered@1.1.1 DONE}
		smooth-scrolling-everywhere-1.1.jar Smooth Scrolling Everywhere {smooth-scrolling-everywhere@1.1 DONE}
		Gobber2-1.14.4-2.1.116.jar Gobber 2 {gobber2@2.1.116 DONE}
		tombstone-4.1.7-1.14.4.jar Corail Tombstone {tombstone@4.1.7 DONE}
		cg_1.14-1.0.jar Crying Ghasts {cg@1.0 DONE}
		Mine and Slash - Expansion-1.4.2.30.jar Mine & Slash Expansion {mineandslashexpansion@1.4.2.30 DONE}
		EnigmaticLegacy-1.6.2.jar Enigmatic Legacy {enigmaticlegacy@1.6.2 DONE}
		noRecipeBook_v1.4.1formc1.14.4.jar No Recipe Book {norecipebook@1.4.1 DONE}
		AppleSkin-mc1.14.4-forge-1.0.13.jar AppleSkin {appleskin@1.0.13 DONE}
		mobcatcher-1.2b.jar Mob Catcher {mobcatcher@1.2b DONE}
		PattysMoreArmor-Stable-1.14.4-1.5.jar pattysmorearmor {pattysmorearmor@1.5 DONE}
		ilikewood-1.14.4-1.1.2.3.jar I Like Wood {ilikewood@1.1.2.3 DONE}
		Aquaculture-1.14.4-2.0.2.jar Aquaculture 2 {aquaculture@1.14.4-2.0.2 DONE}
		XL-Food-Mod-1.14.4-2.1.1.jar XL Food Mod {xlfoodmod@2.1.1 DONE}
		switchbow-1.5.8.jar Switch-Bow {switchbow@1.5.8 DONE}
		FastLeafDecay-v17.jar FastLeafDecay {fastleafdecay@v17 DONE}
		pokecube-aio-1.14.4-2.0.a.jar Thut Core {thutcore@6.0.a DONE, thutcrafts@6.0.a DONE, thut_wearables@6.0.a DONE, pokecube@6.0.a DONE, pokecube_mobs@6.0.a DONE, pokecube_adventures@6.0.a DONE}
		farlanders-1.14.4-1.1.0.jar The Farlanders {farlanders@1.1.0 DONE}
		BetterAdvancements-1.14.4-0.1.0.93.jar Better Advancements {betteradvancements@0.1.0.93 DONE}
	Player Count: 1 / 8; [ServerPlayerEntity['Audalics'/198, l='New World', x=-154.59, y=29.00, z=100.91]]
	Data Packs: vanilla, mod:appleskin, mod:aquaculture, mod:arl, mod:azurecompat, mod:behgameon, mod:betteradvancements (incompatible), mod:betteranimalsplus, mod:bettercaves, mod:biomesoplenty, mod:bluepower, mod:bountifulbaubles, mod:cg, mod:citadel, mod:combat, mod:controlling, mod:cookingforblockheads, mod:craftandhunt, mod:craftingtweaks, mod:cucumber, mod:curios, mod:cyclopscore, mod:doubledoors, mod:druidcraft, mod:easy_steel, mod:edit_sign, mod:enigmaticlegacy, mod:explorercraft, mod:extendedfoods, mod:extendedslabs, mod:extrabows, mod:farlanders, mod:fastleafdecay, mod:flopper, mod:forge, mod:forgeendertech, mod:gemsplusplus, mod:glassential, mod:gobber2, mod:ilikewood, mod:jei, mod:mcwbridges, mod:mgui, mod:mineanddecoration, mod:mineandslashexpansion, mod:mmorpg, mod:mobcatcher, mod:modnametooltip, mod:mousetweaks, mod:moveplus, mod:mysticallib, mod:mysticalworld, mod:nethercraft, mod:norecipebook, mod:oresabovediamonds, mod:paintings, mod:pattysmorearmor, mod:pattysmorestuff, mod:pattysmoretools, mod:peacefulsurface, mod:pickupnotifier, mod:placebo, mod:quark, mod:rediscovered, mod:sereneseasons, mod:simpledivegear, mod:skltlampforthteen, mod:smooth-scrolling-everywhere, mod:spartanshields, mod:spw, mod:switchbow, mod:tetra, mod:theoneprobe, mod:thutcore, mod:tombstone, mod:undergroundbiomes, mod:upgrade_aquatic, mod:uppers, mod:useless_sword, mod:vanillatweaks, mod:wstweaks, mod:wsw, mod:wyrmroost, mod:xaerominimap, mod:xaeroworldmap, mod:xercamod (incompatible), mod:xlfoodmod, mod:yoyos, file/bluepower
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'forge'
[22Dec2019 12:55:32.870] [Client thread/INFO] [STDOUT/]: [net.minecraft.util.registry.Bootstrap:func_179870_a:100]: #@!@# Game crashed! Crash report saved to: #@!@# /home/alexander/.config/GDLauncher/GDLauncher/instances/packs/KISS/crash-reports/crash-2019-12-22_12.55.32-server.txt

```

**Comment by Thutmose** [comment]
This is fixed in the newly updated beta release for 1.14.4
